README.md

```diff
@@ -11,19 +11,40 @@ The VSCode api is composed of:
   - If it's some advanced features that don't make a lot of sense on Monaco (debug, tests...), it just throws an error when you try to use it.
 
 
-To implement by hands the optional features (type hierarchy, call hierarchy...), you can use the `Services` namespace from `vscode/services`:
+To implement by hands the optional features (file system, workspace folders, file...), you can use the `Services` namespace from `vscode/services`:
 ```typescript
 import { Services } from 'vscode/services'
 Services.install({
-   languages: {
-       registerTypeHierarchyProvider (documentSelector, provider) {
-           ...
-       }
-   }
+  workspace: {
+    workspaceFolders: ...,
+    createFileSystemWatcher (documentSelector, provider) {
+      ...
+    },
+    onDidSaveTextDocument
+  },
+  window: {
+    withProgress: ...
+  }
 })
 ```
 
+Also, monaco-editor use `standalone` versions or the vscode services, which are much simpler.
 
+You may want to provide your custom implementations of them, especially for: `textModelService`, `codeEditorService` and `notificationService`. Do do so, you can give them as the third parameter while creating your first editor.
```

Comment:
`Do do so` -> `To do so`
`give them as` -> `provide them as` ?

Reply:
97e2bb7f44c777a8022fc3a7c72d243a1aa78e06